src/core/Form/Combobox/Combobox/Combobox.tsx

```diff
@@ -0,0 +1,595 @@
+      disabledKeys,
+      filterInputValue,
+    } = this.state;
+
+    const focusToMenu = () => {
+      if (
+        this.popoverListRef !== null &&
+        this.popoverListRef.current !== null &&
+        showPopover
+      ) {
+        this.popoverListRef.current.focus();
+      }
+    };
+
+    const {
+      id,
+      className,
+      items: propItems,
+      labelText,
+      onItemSelectionsChange,
+      chipListVisible,
+      chipActionLabel,
+      removeAllButtonLabel,
+      visualPlaceholder,
+      emptyItemsLabel,
+      defaultSelectedItems,
+      onChange: propOnChange,
+      debounce,
+      status,
+      statusText,
+      selectedItems: controlledItems,
+      onItemSelect,
+      onRemoveAll,
+      ...passProps
+    } = this.props;
+
+    const selectedAndDisabledKeys = Object.keys(selectedKeys).reduce(
+      (amount: number, key: string) => {
+        if (key in disabledKeys) {
+          return amount + 1;
+        }
+        return amount;
+      },
+      0,
+    );
+
+    const showChipList =
+      chipListVisible && selectedItems && selectedItems.length > 0;
+
+    const showRemoveAllButton =
+      removeAllButtonLabel &&
+      Object.keys(selectedKeys).length > 0 &&
+      Object.keys(selectedKeys).length !== selectedAndDisabledKeys;
+
+    return (
+      <HtmlDiv
+        role="combobox"
+        aria-haspopup="listbox"
+        aria-controls={`${id}-popover`}
+        aria-expanded={showPopover}
+        {...passProps}
+        className={classnames(baseClassName, className, {
+          [comboboxClassNames.open]: showPopover,
+          [comboboxClassNames.error]: status === 'error',
+        })}
+      >
+        <HtmlDiv className={classnames(comboboxClassNames.wrapper, {})}>
+          <Debounce waitFor={debounce}>
+            {(debouncer: Function) => (
+              <FilterInput
+                id={id}
+                labelText={labelText}
+                items={propItems}
+                onFilter={(filtered) =>
+                  this.setState({ filteredItems: filtered })
+                }
+                filterFunc={filter}
+                forwardedRef={this.filterInputRef}
+                onFocus={() => setPopoverVisibility(true)}
+                onKeyDown={handleKeyDown}
+                onBlur={handleBlur}
+                value={filterInputValue}
+                onChange={(value: string | undefined) => {
+                  if (propOnChange) {
+                    debouncer(propOnChange, value);
+                  }
+                  this.setState({ filterInputValue: value });
+                }}
+                visualPlaceholder={visualPlaceholder}
+                status={status}
+                statusText={statusText}
+              />
+            )}
+          </Debounce>
+          <Popover
+            sourceRef={this.filterInputRef.current}
+            matchWidth={true}
+            id={`${id}-popover`}
+            portalStyleProps={{ backgroundColor: 'white' }}
```

Comment:
Is there a better way for handling these? Popover is probably implemented this way currently, but using inline js styles is not the best way to go.

Reply:
This `portalSyleProps` might be leftover from the popover example. 
Might not be even needed in this case to give it background color (assuming popover  has `white` colour by default)

Reply:
If it doesn't we might have to use inline styles, as the portal is not a descendant of the actual component => styles are not necessarily applied correctly when nested.

Reply:
If it works and is required, I'd prefer something like `.fi-popover { ...`

Reply:
It seems that without giving the `color` for  `portalStyleProps` the colour will be transparent by default.

The easiest solution seems to be that I give white background color for the `.fi-combobox-item-list`.